lib/components/app/app-menu.tsx

```diff
@@ -123,46 +120,23 @@ class AppMenu extends Component<
         const label =
           localizedLabel === localizationId ? configLabel : localizedLabel
 
-        if (children) {
-          return (
-            <Fragment key={id}>
-              <MenuItem
-                className="expansion-button-container menu-item expand-submenu-button"
-                id={id}
-                onSelect={() => this._toggleSubmenu(id)}
-              >
-                <IconAndLabel
-                  iconType={iconType}
-                  iconUrl={iconUrl}
-                  label={label}
-                />
-                <StyledIconWrapper className="expand-menu-chevron">
-                  {isSubmenuExpanded ? <ChevronUp /> : <ChevronDown />}
-                </StyledIconWrapper>
-              </MenuItem>
-              <AnimateHeight
-                duration={500}
-                height={isSubmenuExpanded ? 'auto' : 0}
-              >
-                <div className="sub-menu-container">
-                  {this._addExtraMenuItems(children)}
-                </div>
-              </AnimateHeight>
-            </Fragment>
-          )
-        }
-
         return (
           <AppMenuItem
             className={subMenuDivider ? 'app-menu-divider' : undefined}
             href={href}
-            icon={<Icon iconType={iconType} iconUrl={iconUrl} />}
+            icon={
+              typeof iconType === 'string' ? (
```

Comment:
This did not seem to work with some entries in the ATL config.

Reply:
See screenshot: 
<img width="324" alt="image" src="https://user-images.githubusercontent.com/56846598/213574092-7c148b71-4896-407c-b107-baa15cc41c0d.png">